packages/tokens/src/fonts.ts

```diff
@@ -0,0 +1,6 @@
+const fonts = {
+  default: `Akzidenz, 'Helvetica Neue', Helvetica, Arial, sans-serif`,
+  code: `'Source Code Pro', monospace`,
+};
+
+export default fonts;
```

Comment:
Can we call this `fontFamilies` instead? These are not comprehensive font declarations since they are only meant to be used with the `font-family` CSS rule.

Reply:
👍